internal/engine/wazevo/backend/isa/amd64/instr.go

```diff
@@ -1944,10 +2097,12 @@ var defKinds = [instrMax]defKind{
 	shiftR:                 defKindNone,
 	imm:                    defKindOp2,
 	unaryRmR:               defKindOp2,
+	xmmRmiReg:              defKindOp2,
```

Comment:
shouldn't this be defineNone just like other arithmetics that overrides one operand?